relays/utils/src/metrics.rs

```diff
@@ -61,11 +75,64 @@ pub trait StandaloneMetrics: Metrics {
 	}
 }
 
-impl Default for MetricsParams {
+impl Default for MetricsAddress {
 	fn default() -> Self {
-		MetricsParams {
+		MetricsAddress {
 			host: "127.0.0.1".into(),
 			port: 9616,
 		}
 	}
 }
+
+impl MetricsParams {
+	/// Creates metrics params so that metrics are not exposed.
+	pub fn disabled() -> Self {
```

Comment:
`disabled()` should be used instead of `default()`, I guess.

Reply:
Yes, or `From<Option<MetricsAddress>>` - `MetricsAddress` is used in cli